### PR TITLE
fixes #7990 - test package installation using katello-agent

### PR DESCRIPTION
**Pull request by stbenjam**
Needs https://github.com/Katello/katello-agent/pull/19

### CODE REVIEW

File: bats/fb-content-katello.bats [hunk 1]
```diff
@@ -124,8 +124,22 @@ EOF
...
 }
 
-@test "install package from content view" {
-  tPackageInstall walrus && tPackageExists walrus
+@test "install katello-agent" {
+  tPackageInstall katello-agent && tPackageExists katello-agent
+}
+
+@test "start katello-agent" {
+  service goferd start
```
Comment (ehelms) [inline bats/fb-content-katello.bats]: I believe this *should* be started automatically (https://github.com/Katello/katello-agent/blob/master/katello-agent.spec#L52) -- so maybe we should do a check that it is running?

**Comment by ehelms** [comment]
Not necessary for this particular set of tests, but checking that an applicable errata is available might be good when doing agent install tests. Should we make this an optional set of tests similar to --content? --remote-actions?

**Comment by stbenjam** [comment]
Yea, that sounds good to have more coverage on errata.  

I wonder if we could ask the Pulp team to give us a more realistic test repo for errata, https://repos.fedorapeople.org/repos/pulp/pulp/demo_repos/test_errata_install uses `enhancements` as the errata type (which is not a type RHEL/Fedora/etc use)

It'd be nice if it'd look more like a RHEL/Fedora repo using bugfix/enhancement/security/newpackage.

**Comment by ehelms** [comment]
We just work on making our own similar to the zoo repo - @jlsherrill had a modified one that he uses that includes an errata. But if you look at the create script, Zoo was created using our own tools -- https://lzap.fedorapeople.org/fakerepos/zoo5/

**Comment by ehelms** [comment]
@stbenjam bump

**Comment by ehelms** [comment]
@stbenjam thoughts on this?

**Comment by stbenjam** [comment]
@ehelms Updated - will do errata test separately (filed http://projects.theforeman.org/issues/12286)

**Comment by ehelms** [comment]
ACK -- we'll see what the pipeline thinks of this

**Pull request merged**